### ISSUE TITLE
Show average again

### ISSUE THREAD

**Issue by ThomasZitzler** [cschleiden/azure-boards-estimate#37]
In V1 there was a field showing the average vote, this was quite helpful.
Can you please add that back again? 

Maybe you can round that value so that it can also be accepted as estimate?

**Comment by cschleiden** [comment]
For V1 I also received feedback that the average was not the right way to display it. Did you usually use it? 
Would #31 help here?

**Comment by ThomasZitzler** [comment]
Yes, we usually picked a rounded version of the average. Just showing would also help as you can enter a value.

**Comment by pabrams** [comment]
Most guidance says not to take the average.  should use the nearest discrete value to the average.  I suppose displaying it wouldn't hurt anything though.

**Comment by RafaelKoch** [comment]
We are using the average (not even rounded) quite heavily and with reasonable success in planning and later also reporting. This is especially useful when a sprint has to adress a high number of small bugs. Planning is easily thrown off when going to the nearest majority vote. All not out-of-the-book but also not uncommon.

So the behavior of V1 was just right for us. I can live with typing it, but displaying the average is highly desirable.

**Comment by aheavin** [comment]
@cschleiden - Yes we used average the same as @pabrams to use the nearest value. Please bring this back into the display.

**Comment by bdmurray** [comment]
Agree with this thread. Maybe two buttons for Nearest Majority and Average to give people options?

**Comment by jrlilly** [comment]
We've used the average as well.  Having to type would be fine, it's just helpful to not have to calculate it.

**Comment by scottloomis** [comment]
@cschleiden - I think #31 could help.  What was particularly helpful for us was just the automatic average to see where we aligned as a group without manually computing the average.

We have a discussion if things were estimated vastly different 1 vs. 8 , but if there are 6 people estimating and we have three 3 cards and three 2 cards it just makes it more streamlined experience to have it automatically tell us the average.  We also always round(up/down) to the nearest card value based upon the average

Seems like a show average/show number of people may be a nice option to put in the settings.

**Comment by Nardax** [comment]
Please bring back the average as a value that can be automatically added to the card. 

Why is the average important?

Let's say that the vote is four "3"s and two "5"s. We have four possible choices:

- take the 3s and tell the people that voted 5 that there vote didn't count, or worse that they are wrong
- take the 5s and tell the people that voted 3 that we are "rounding up"
- force everyone to vote again until they all come to the same number. This will be an epic waist of time and may cause a mutiny.
 - take the **straight average** (3.7) and let everyone's vote count equally. 

I coach teams to vote once and look for everyone's vote to land in a consecutive series of 3 cards (e.g. 2-3-5, or 5-8-13). If the vote lands with everyone contained in a series of three cards then take the average and move on. If the vote isn't in a series of three cards (e.g. 3-5-13, skipped 8) then the high and low values discuss why they are high or low and we vote again. After voting a second time take the straight average and move on, no additional discussion. Story points are supposed to be estimates not exacts values. Teams will normalize over time so continuous revoting only serves to create groups of people who think they are wrong and groups of people who think they are right. The whole team needs to feel that they are taking ownership of the story and not just the members of the team that voted for the chosen number.
